### PR TITLE
Add a service to unsubscribe all Brexit Readiness Finder emails

### PR DESCRIPTION
**Pull request by koetsier**
and deactiveate all subscriptions.

trello: 

https://trello.com/c/ChmNnVRZ/523-remove-all-business-readiness-finder-subscriptions
https://trello.com/c/EgTYD0qt/497-send-a-message-to-subscribers-to-the-business-readiness-finder

### CODE REVIEW

File: app/services/unsubscribe_brf_service.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
+class UnsubscribeBRFService
+  MESSAGE = <<~BODY.freeze
+    goes here!
```
Comment (huwd) [inline app/services/unsubscribe_brf_service.rb]: Needs a body

File: app/services/unsubscribe_brf_service.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  BODY
+
+  SUBJECT = "subject goes here".freeze
```
Comment (huwd) [inline app/services/unsubscribe_brf_service.rb]: needs a subject

File: app/services/unsubscribe_brf_service.rb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  def self.call
+    UnsubscribeBRFService.new.call
+  end
+
+  def call
+    subscriber_lists = SubscriberList.where("title like ?", "%Brexit guidance for your business%")
+    puts "Unsubscribing #{subscriber_lists.count} subscriptions"
+    subscriber_lists.inject(false) do |result, subscriber_list|
```
Comment (huwd) [inline app/services/unsubscribe_brf_service.rb]: This is cool,
So do i understand this correnctly `.inject(false)` takes the enumberable `subscriber_lists` and sets a default value of false for it... then overwrites that only if the block returns true?

Comment (koetsier) [reply app/services/unsubscribe_brf_service.rb]: it's a way of or-ing all the results so we know if a message has been sent at all

**Comment by koetsier** [comment]
- Need to unique the subscribers
- Update the content

**Pull request closed without merging**